packages/scandipwa/src/component/WishlistItem/WishlistItem.container.js

```diff
@@ -158,6 +159,41 @@ export class WishlistItemContainer extends PureComponent {
             );
     }
 
+    recalculatePrice() {
+        const {
+            product: {
+                type_id,
+                items
+            }
+        } = this.props;
+
+        if (type_id !== GROUPED || !items) {
+            return;
+        }
+
+        const { product: { price_range: { minimum_price } } } = this.props;
+
+        items.forEach((link) => {
```

Comment:
"Link" is not informative. Consider renaming to `item` or `variant` or something else.

Reply:
Renamed to item